src/qmflows/packages/cp2k_package.py

```diff
@@ -8,11 +8,13 @@
 
 from scm import plams
 
+from .packages import Package, Result, package_properties, parse_output_warnings
 from ..parsers.cp2KParser import parse_cp2k_warnings
 from ..settings import Settings
-from ..warnings_qmflows import cp2k_warnings
-from .packages import (Package, Result, package_properties,
-                       parse_output_warnings, WarnMap)
+from ..warnings_qmflows import cp2k_warnings, Key_Warning
+from ..type_hints import WarnMap
+
+__all__ = ['cp2k']
 
 # ====================================<>=======================================
 charge_dict: Dict[str, int] = {
```

Comment:
I am going to replace this dictionary with [mendeleev](https://pypi.org/project/mendeleev/)

Reply:
Shall I remove it in this branch?

Reply:
I did it on my branch :)